smos-docs-site/pages/configuration-reference.markdown

```diff
@@ -0,0 +1,89 @@
+---
+title: Customisation Reference
+date: 2020-04-07
+---
+
+# Misc #
+
+contexts
+
+: ???
+
+work-filter
+
+: ???
+
+# Customizing directories #
+
+archive-dir
+
+: the archive directory
+
+workflow-dir
+
+: the workflow directory
+
+projects-dir
+
+: the projects directory
+
+archived-projects-dir
+
+: the archive projects directory
+
+
+# Customising keybindings #
+
+<!-- NOTE: It should be nice to be able to generate the reference for
+           the keybingings automatically, since the information are
+           already present in the help screen -->
+
+Keybindings in smos are context sensitive.
+
+The generic syntax for keybindings is:
+
+```yaml
+keys:
+    context:
+        subcontext:
+        - action: actionName
+          key: <Esc>
+```
+
+The following paragraph lists the valid contexts and the valid
+keybindings.
```

Comment:
I'm impressed with how far you've gotten with this.

I've made #40 to track the automated generation of the keybinding config.